packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/image-toolbar-button.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { withRef } from '@udecode/cn'
+
+import { Icons } from './icons'
+
+import { ToolbarButton } from './toolbar'
+import { useEditorState } from '@udecode/plate-common'
+import { helpers } from '../../plugins/core/common'
+import { ELEMENT_IMG, insertImg } from '../../plugins/create-img-plugin'
+import { useCMS } from '@toolkit/react-core'
+
+const useImageToolbarButtonState = () => {
+  const editor = useEditorState()
+
+  const isBlockActive = () => helpers.isNodeActive(editor, ELEMENT_IMG)
+
+  return {
+    pressed: isBlockActive(),
+  }
+}
+
+const useImageToolbarButton = (state) => {
+  const editor = useEditorState()
+  const cms = useCMS()
+
+  const onMouseDown = (e) => {
+    e.preventDefault()
+
+    cms.media.open({
+      allowDelete: true,
+      directory: '', //TODO: get actual directory?
```

Comment:
**question**: Is this TODO worth a PBI?

Reply:
@jeffsee55 - I'm not sure what this is used for (copied it over from the previous implementation)